admin-base/core/src/main/java/com/peregrine/admin/servlets/AdminPaths.java

```diff
@@ -40,4 +40,5 @@
     public static final String RESOURCE_TYPE_LIST_VERSIONS = API_PREFIX + "listVersions";
     public static final String RESOURCE_TYPE_RESTORE_RECYCLABLE = API_PREFIX + "restoreRecyclable";
     public static final String RESOURCE_TYPE_USER_PREFERENCES = API_PREFIX + "userPreferences";
+    String RESOURCE_TYPE_IS_SITE_NAME_AVAILABLE = API_PREFIX + "sites/name/available";
```

Comment:
any reason not to follow suit and making this constant value `public static final` as the others?

Reply:
Ideally I'd remove the others :) as they are not necessary, but let me simplify the PR and just add the modifiers to the new field for now.